portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path

```diff
@@ -77,6 +77,31 @@
 	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Modified Date')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FILTER_BY_SITE_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Site or Asset Library')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_BY_STATUS_ANY_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Any')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_BY_STATUS_APPROVED_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Approved')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_BY_STATUS_DRAFT_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Draft')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_BY_STATUS_SCHEDULED_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Scheduled')]</td>
+	<td></td>
+</tr>
```

Comment:
All of these paths look like the same ... so, change in the test you pass what you want and use one unique variable FILER_BY_blablabla (find a good name) and use something like...
`//*[contains(@class, 'dropdown-item') and contains(text(),'${key_filterBySomething}')]`